Terraform_Script_Create_WP/main.tf

```diff
@@ -114,8 +113,8 @@ resource "azurerm_linux_web_app" "wordpress_web_app" {
     always_on              = var.app_service_alwayson
     vnet_route_all_enabled = true
     application_stack {
-      docker_image     = var.wordpress_container_linux_fx_version
-      docker_image_tag = "latest"
+      docker_image_name   = "${var.wordpress_container_linux_fx_version}:latest"
```

Comment:
please use 8.2 instead of latest tag. latest tag is deprecated, and it corresponds to 8.0 php.